tasks/centos.yml

```diff
@@ -38,20 +38,31 @@
     - lv.1.create is defined
     - lv.1.create|bool
 
-- name: centos | checking for scsi devices
-  command: sg_scan
-  become: true
-  register: scsi_devices
-  changed_when: false
+- block:
+  - name: centos | installing sg3_utils
+    package:
+    name: sg3_utils
+    state: present
+    become: true
 
-- name: centos | rescanning for new disks
-  command: /usr/bin/rescan-scsi-bus.sh
-  become: true
-  changed_when: false
-  when: scsi_devices.stdout|length > 0
+  - name: centos | checking for scsi devices
+    command: sg_scan
```

Comment:
Is there anything in the output of this that can help us?

Reply:
Not really:
# sg_scan -v
Unable to open: /dev/sg0, errno=2
Unable to open: /dev/sg1, errno=2
Unable to open: /dev/sg2, errno=2
Unable to open: /dev/sg3, errno=2